lxc_privilege_converter.sh

```diff
@@ -141,6 +141,25 @@ find_next_free_id() {
     pvesh get /cluster/nextid
 }
 
+# select ID for the new container
+select_container_id() {
+    next_free_id=$(find_next_free_id)
+    while true; do
+	read -r -p "Select the new container ID [$next_free_id]: " NEW_CONTAINER_ID
+
+	if [ -z "$NEW_CONTAINER_ID" ]; then
+	    NEW_CONTAINER_ID=$next_free_id
+	    break
+	fi
+
+	if pct list | grep -q "^$NEW_CONTAINER_ID "; then
+	    echo "Already used container ID. Please try again."
```

Comment:
I realize that this check isn't perfect, because it is only checking local containers. I only have a single PVE system, so I haven't learned about clusters yet. Also, I suppose VM IDs need to be checked as well. (Just thought of that now.) I think an imperfect check is better than nothing.